docs/core-concepts/working-with-templates/responsive-images.md

```diff
@@ -57,9 +57,20 @@ module.exports = {
 
 ```django
 {# lib/modules/my-image-widget/views/widget.html #}
-<img src="{{ apos.attachments.url(data.widget._image.attachment, { size: data.options.size or 'full' }) }}" srcset="{{ apos.images.srcset(data.widget._image.attachment) }}" sizes="{{ data.options.sizesAttr or '100vw' }}" alt="{{ data.widget._image.description or data.widget._image.title }}">
+  <img 
+    src="{{ apos.attachments.url(data.widget._image.attachment, { size: data.options.size or 'full' }) }}" 
+    srcset="{{ apos.images.srcset(data.widget._image.attachment) }}" 
+    sizes="{{ data.options.sizesAttr or '100vw' }}" 
+    alt="{{ data.widget._image.description or data.widget._image.title }}"
+   >
 ```
 
+The `apos.images.srcset` helper even offers cropping. If you add `relationship` you can use `apostrophe-attachment` cropping functions effectively.
```

Comment:
Is this really necessary? It is not necessary for the `src` attribute. For convenience the relationship is automatically propagated to a property of the attachment as long as it is found properly with `images.first` or similar, and then `attachments.url` automatically knows what to do with it. Unless this dance is coming apart somewhere in `srcset`.

Reply:
This isn't incorrect, but you're right, Tom. Using `apos.images.first` would packaged this information up for the user. Unfortunately we aren't using that for most of the examples here. _Fortunately_ this is the _only_ place in the docs where we get the image attachment this way, rather than using the `first()` method.

@felixlberg Would you change this example to use that method instead? The note would then say something like "the `apos.images.srcset` maintains cropping when you pass in an image object using `apos.images.first()` or `apos.images.all()`". Thanks for the contribution.

Reply:
The best place to explain the second argument to `srcset()` would be in the apostrophe-images module reference. Otherwise we want to guide people to the _best_ way to do something.